purchase_request/views/purchase_request_view.xml

```diff
@@ -0,0 +1,494 @@
+                />
+                <filter
+                    name="assigned_to_me"
+                    domain="[('assigned_to','=', uid)]"
+                    help="Assigned to me"
+                />
+                <filter
+                    name="my_requests"
+                    domain="[('requested_by','=', uid)]"
+                    help="My requests"
+                />
+                <field name="requested_by" />
+                <field name="assigned_to" />
+                <field name="picking_type_id" />
+                <group expand="0" string="Group By...">
+                    <filter
+                        name="requested_by"
+                        string="Requested by"
+                        icon="fa-user"
+                        domain="[]"
+                        context="{'group_by':'requested_by'}"
+                    />
+                    <filter
+                        name="assigned_to"
+                        string="Assigned to"
+                        icon="fa-user"
+                        domain="[]"
+                        context="{'group_by':'assigned_to'}"
+                    />
+                    <filter
+                        name="source"
+                        string="Source"
+                        icon="fa-caret-square-o-left"
+                        domain="[]"
+                        context="{'group_by':'origin'}"
+                    />
+                    <filter
+                        name="start_date"
+                        string="Start Date"
+                        icon="fa-calendar"
+                        domain="[]"
+                        context="{'group_by':'date_start'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="purchase_request_form_action">
+        <field name="name">Purchase Requests</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">purchase.request</field>
+        <field name="view_mode">tree,form</field>
+        <field name="context">{"search_default_requested_by":uid}</field>
+        <field name="search_view_id" ref="view_purchase_request_search" />
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Click to start a new purchase request process.
+            </p>
+            <p>
+                A purchase request is an instruction to Purchasing to procure
+            a certain quantity of materials services, so that they are
+            available at a certain point in time.
+            </p>
+        </field>
+    </record>
+    <menuitem
+        id="menu_purchase_request_pro_mgt"
+        parent="purchase.menu_procurement_management"
+        action="purchase_request_form_action"
+    />
+    <menuitem
+        id="parent_menu_purchase_request"
+        name="Purchase Requests"
```

Comment:
```suggestion
        name="Purchase Requests"
        sequence="22"
```
Just my request. Normally, I we would want menu "Purchase Request" to be right before "Purchase", but currently with some other module install, it might not be.

![Selection_003](https://user-images.githubusercontent.com/1973598/140917840-c413c45b-4641-4820-85ad-eff874d82e63.png)

Reply:
@kittiu i believe that what you are saying it's okay but is not really important. It is quite subjective. For me it is okay both ways